install/streams.csv

```diff
@@ -2498,11 +2498,11 @@ WVZ,10,1,false,false,false,2003-09-06T00:00:00Z,9999-01-01T00:00:00Z
 WVZ,10,50,false,false,true,1990-05-16T14:12:00Z,2003-09-05T23:51:00Z
 WVZ,10,100,false,false,false,2003-09-06T00:00:00Z,9999-01-01T00:00:00Z
 WVZ,11,50,false,false,true,2003-09-06T10:56:00Z,2003-12-11T16:33:00Z
-WVZ,20,1,false,false,false,2003-09-06T00:00:00Z,2011-10-10T00:00:00Z
-WVZ,20,50,false,false,false,2011-10-10T00:00:05Z,2020-01-15T03:38:00Z
```

Comment:
Like other stations, it's weird that for a long time (from 2003 to 2011), only 1hz was streaming & there's no 50hz data.